sass/abstracts/_variables.scss

```diff
@@ -6,6 +6,10 @@ $color-primary-dark-1: #0b7a00;
 $color-primary-dark-2: #085c00;
 
 $color-primary-light-1: #56b84d;
+$color-primary-light-2: #04a312;
+$color-primary-light-1-trans: #56b84d2e;
```

Comment:
Dummy question: what  `trans` stands for? Transformation?

Reply:
It stands for "transparent". Maybe I should put a comment in _variables/ to separate them from the others and make more clear. What do you think?

Reply:
No need it is clear now.